pymc/stats/model_comparison.py

```diff
@@ -0,0 +1,126 @@
+#   Copyright 2022 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+from typing import Optional, Sequence
+
+from arviz import InferenceData, dict_to_dataset
+from fastprogress import progress_bar
+
+import pymc
+
+from pymc.backends.arviz import _DefaultTrace
+from pymc.model import Model, modelcontext
+from pymc.util import dataset_to_point_list
+
+__all__ = ("compute_log_likelihood",)
+
+
+def compute_log_likelihood(
+    idata: InferenceData,
+    *,
+    var_names: Optional[Sequence[str]] = None,
+    extend_inferencedata: bool = True,
+    model: Optional[Model] = None,
+    sample_dims: Sequence[str] = ("chain", "draw"),
+    progressbar=True,
+):
+    """Compute elemwise log_likelihood of model given InferenceData with posterior group
+
+    Parameters
+    ----------
+    idata: InferenceData
```

Comment:
```suggestion
    idata : InferenceData
```

colons need a space both before and after

Reply:
@MarcoGorelli Any chance we can add this to pre-commit? It's unintuitive :)

Reply:
hey 👋 yeah I think pydocstyle might catch this?

Reply:
> hey wave yeah I think pydocstyle might catch this?

Thanks! @OriolAbril Do you think it would be feasible?

Reply:
Any help on this will be welcome